src/pages/Home/subcomponents/CitySuggestions/SuggestedCity.js

```diff
@@ -1,54 +1,74 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
 
-export default function SuggestedCity({ city, handleCityClick }) {
+export default function SuggestedCity({
+  city,
+  handleCityClick,
+  totalCityCount,
+  setTotalCityCount,
+  index
+}) {
   const [clicked, handleClick] = useState(false);
   const [removed, handleRemoved] = useState(false);
   function handleCityClickHelper(city) {
     handleClick(true);
     handleCityClick(city);
     setTimeout(() => handleRemoved(true), 2000);
```

Comment:
```suggestion
    setTimeout(() => {
       handleRemoved(true);
       setTotalCityCount(totalCityCount - 1);
    }, 2000);
```
Apologies for not doing this in the first place! This is necessary for the checkmark svg to finish running for the last city.